lib/internal/url.js

```diff
@@ -930,6 +957,30 @@ class URL {
   toJSON() {
     return this[kFormat]({});
   }
+
+  static createObjectURL(obj) {
+    const blob = lazyBlob();
+    if (!blob.isBlob(obj))
+      throw new ERR_INVALID_ARG_TYPE('obj', 'Blob', obj);
+
+    const id = lazyCryptoRandom().randomUUID();
+
+    storeDataObject(id, obj[blob.kHandle], obj.size, obj.type);
+
+    return `blob:node:${lazyThreadId()}:${id}`;
```

Comment:
Actually, why are we adding the thread id here?

Reply:
Removed